src/fsevents.c

```diff
@@ -6,6 +6,8 @@
 #include "runner.h"
 #include <node_api.h>
 
+#define CONST(name, const_val) CHECK(napi_create_int32(env, const_val, &value) == napi_ok); CHECK(napi_set_named_property(env, constants, name, value) == napi_ok);
```

Comment:
Not wrong but it's good style to wrap multi-line macros in a do/while block. If you wonder why:
```c
if (cond) CONST("foo", foo);  // breaks because the second line is evaluated unconditionally
```
Aside: you can shorten it to this:
```c
#define CONST(name) \
  do { \
    CHECK(napi_create_int32(env, name, &value) == napi_ok); \
    CHECK(napi_set_named_property(env, constants, #name, name) == napi_ok); \
  } while (0)
```